TODO/TODO/settings.py

```diff
@@ -124,7 +125,11 @@
 # ]
 
 
-AUTHENTICATION_BACKENDS = ['authuser.backends.EmailBackend']
+AUTHENTICATION_BACKENDS = [
+    'authuser.backends.EmailBackend',
+    'django.contrib.auth.backends.ModelBackend'
```

Comment:
Check how does it work with multiple authentication backends.

Reply:
**Backends authentication working principle:**
- mulltiple backends can be used in a tandem 
- is authenticated on the first backend check if found